SU-SWS/ood-giving-campaign
# GIVCAMP-365 | Homepage Changemaker section editable fields

# READY FOR REVIEW

# Summary
- Make more of the Changemaker section on the homepage editable
- Note: this was the original idea - but currently, client decided to move the "Changemaker section higher up on the page so we ended up using a generic Section component to build out the Changemaker section. However, we'd still like to merge this in for it to be editable in the future.

# Review By (Date)
- Retro

# Criticality
- 2

# Review Tasks

## Setup tasks and/or behavior to test

1. go to Storyblok `Test/Yvonne Test Homepage Background Blend`
2. Choose PR 309 from the visual editor
3. Scroll down to the changemaker section to see the editable fields

![Screenshot 2024-07-09 at 5 53 12 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/74e2e618-a678-4cba-9eaa-1baa66969811)

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-365
